### PR TITLE
Daemon support for community seed

### PR DESCRIPTION
**Pull request by sandipbhoir**
## Description
This PR will cover core part of issue: https://github.com/Zilliqa/Issues/issues/501

## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [x] small-scale cloud test

### CODE REVIEW

File: daemon/ZilliqaDaemon_AWS.cpp [hunk 1]
```diff
@@ -25,16 +25,24 @@
...
+#include <thread>
 #include <unordered_map>
 #include <vector>
 
 using namespace std;
+namespace po = boost::program_options;
+
+#define SUCCESS 0
+#define ERROR_IN_COMMAND_LINE -1
+#define ERROR_UNHANDLED_EXCEPTION -2
```
Comment (ckyang) [inline daemon/ZilliqaDaemon_AWS.cpp]: Seems `ERROR_UNHANDLED_EXCEPTION` never be used. Is it exception need to be handled?

Comment (KaustubhShamshery) [reply daemon/ZilliqaDaemon_AWS.cpp]: Prefer inline constexpr functions to function macros. (from coding guideline)

Comment (sandipbhoir) [reply daemon/ZilliqaDaemon_AWS.cpp]: Yeah. Fixed.

Comment (KaustubhShamshery) [reply daemon/ZilliqaDaemon_AWS.cpp]: still see macros.

**Pull request merged**